packages/cookies/src/response-cookies.ts

```diff
@@ -14,14 +14,20 @@ export class ResponseCookies {
 
   constructor(responseHeaders: Headers) {
     this._headers = responseHeaders
-    // @ts-expect-error See https://github.com/whatwg/fetch/issues/973
-    const headers = this._headers.getAll('set-cookie')
 
-    for (const header of headers) {
-      const parsed = parseSetCookieString(header)
-      if (parsed) {
-        this._parsed.set(parsed.name, parsed)
-      }
+    const setCookie =
+      // @ts-expect-error See https://github.com/whatwg/fetch/issues/973
+      responseHeaders.getAll?.('set-cookie') ??
+      responseHeaders.get('set-cookie') ??
```

Comment:
**praise:** Thanks for removing this hard dependency on `getAll()`!! I banged onto it while working on isomorphic, but I had to stop for changing priorities. Thanks a million!